src/main/scala/com/wavesplatform/transaction/protobuf/PBTransactionParser.scala

```diff
@@ -0,0 +1,19 @@
+package com.wavesplatform.transaction.protobuf
+import com.wavesplatform.transaction.TransactionParser
+
+import scala.reflect.ClassTag
+import scala.util.Try
+
+trait PBTransactionParser extends TransactionParser.OneVersion {
```

Comment:
Why not just define an object?

Also there's no need to have a `TransactionParser` implementation for PB transactions, because they won't be used in old contexts like network messages or blocks.